### PR TITLE
Web: update packages

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
**What this PR does / why we need it**:
Update packages before we start the code freeze.

Turns out monaco-editor has this known bug https://github.com/microsoft/monaco-editor/issues/4647 starting from 0.50.x
**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by ahmedwaleedmalik** [comment]
/hold
Need to fix failures in CI it seems

**Review by ahmadhamzh — APPROVED** [review]
/approve

**Comment by Waseem826** [comment]
It would be great if we can also fix https://github.com/kubermatic/dashboard/security/dependabot/90 security vulnerability in this PR because dependabot seems to be struggling.

**Comment by ahmedwaleedmalik** [comment]
> It would be great if we can also fix [Server-Side Request Forgery in axios](/kubermatic/dashboard/security/dependabot/90) security vulnerability in this PR because dependabot seems to be struggling.

Sure thing, I'll get back to this PR in a day or two but will go through and fix the security vulns as well.

**Review by ahmadhamzh — APPROVED** [review]
/approve

File: modules/web/.npmrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
-engine-strict=true
+engine-strict=false
```
Comment (ahmedwaleedmalik) [inline modules/web/.npmrc:1]: Just so that dependabot can be functional again.

**Review by Waseem826 — APPROVED** [review]
/approve

**Pull request merged**